lib/hydra/works/models/concerns/generic_file/contained_files.rb

```diff
@@ -10,6 +10,9 @@ module Hydra::Works::GenericFile::ContainedFiles
     directly_contains_one :original_file, through: :files, type: ::RDF::URI("http://pcdm.org/use#OriginalFile"), class_name: "Hydra::PCDM::File"
     directly_contains_one :thumbnail, through: :files, type: ::RDF::URI("http://pcdm.org/use#ThumbnailImage"), class_name: "Hydra::PCDM::File"
     directly_contains_one :extracted_text, through: :files, type: ::RDF::URI("http://pcdm.org/use#ExtractedText"), class_name: "Hydra::PCDM::File"
+    directly_contains_one :mp3, through: :files, type: ::RDF::URI("http://pcdm.org/use#ServiceFile"), class_name: "Hydra::PCDM::File"
```

Comment:
Can you talk about why we're changing the default derivatives to include these four new ones?  I ask because of the comment on line 4 -- I'm not sure if it's meant to be normative or descriptive.